themes/demo/pages/ajax.htm

```diff
@@ -44,16 +49,15 @@ <h1>AJAX framework</h1>
             <h3 class="panel-title">Calculator</h3>
         </div>
         <div class="panel-body">
-            <form role="form" class="form-inline" data-request="onTest" data-request-update="calcresult: '#result'">
-                <input type="text" class="form-control" value="15" name="value1" style="width:100px">
-                <select class="form-control" name="operation" style="width: 70px">
+            <form role="form" class="form-inline" data-request="onTest" data-request-update="calcresult: '#result'" data-request-flash>
+                <input type="text" class="form-control" value="15" name="value1" style="width:100px" data-track-input>
+                <select class="form-control" name="operation" style="width: 70px" data-track-input>
                     <option>+</option>
                     <option>-</option>
                     <option>*</option>
                     <option>/</option>
                 </select>
-                <input type="text" class="form-control" value="5" name="value2" style="width:100px">
-                <button type="submit" class="btn btn btn-primary" data-attach-loading>Calculate</button>
+                <input type="text" class="form-control" value="5" name="value2" style="width:100px" data-track-input>
```

Comment:
Should probably keep the calculate button anyways right?

Reply:
I suppose, but it is redundant at this point.